src/gates.jl

```diff
@@ -1,3 +1,10 @@
+export 𝕀, sx, sy, sz, qft, hadamard, grover
+
+sx = ComplexF64[0 1; 1 0]
+sy = ComplexF64[0 1im; -1im 0]
+sz = ComplexF64[1 0; 0 -1]
+𝕀(::Type{T}, dim=2) where T<: Number = Matrix{T}(I, dim, dim)
```

Comment:
This should be deprecated in new `julia` versions. `I` works

Reply:
Fixed